ui/src/overview.tsx

```diff
@@ -226,7 +226,7 @@ function ParticipantsGroup(props: {
     }
 
     return (await source.filterCount(filter))[0].count;
-  }, [underlay, cohort]);
+  }, [underlay, cohort.id, cohort.name, cohort.underlayName, props.group]);
```

Comment:
So if any of these properties change, then you don't update the counts? Why do you need both `underlay` and `cohort.underlayName`?

Reply:
Good catch. There isn't actually a dependency on `underlay` so it shouldn't be there.